src/bootstrap/src/core/build_steps/tool.rs

```diff
@@ -692,6 +692,80 @@ impl Step for RustAnalyzerProcMacroSrv {
     }
 }
 
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct LlvmBitcodeLinker {
+    pub compiler: Compiler,
+    pub target: TargetSelection,
+    pub extra_features: Vec<String>,
+}
+
+impl Step for LlvmBitcodeLinker {
+    type Output = PathBuf;
+    const DEFAULT: bool = true;
+    const ONLY_HOSTS: bool = true;
+
+    fn should_run(run: ShouldRun<'_>) -> ShouldRun<'_> {
+        let builder = run.builder;
+        run.path("src/tools/llvm-bitcode-linker").default_condition(
+            builder.config.extended
+                && builder.config.tools.as_ref().map_or(
+                    false || builder.build.unstable_features(),
+                    |tools| {
+                        tools.iter().any(|tool| match tool.as_ref() {
+                            "clippy" => "llvm-bitcode-linker" == "clippy-driver",
```

Comment:
This was an accident I guess

Reply:
This is expanded from the macro it previously was generated from. I don't know if it was important for something so I didn't dare to remove it. 

Are you saying I should remove it?

Reply:
It should be indeed removed. `"clippy" => "llvm-bitcode-linker" == "clippy-driver"` has no purpose. Something like `|tools| tools.iter().any(|tool| tool == "llvm-bitcode-linker")` should be enough.

Reply:
I actually read the code now, and it is indeed meaningless. Should be fixed now